modules/tests/test/disk_virt_libguestfs_tasks_test.go

```diff
@@ -42,10 +42,19 @@ var _ = Describe("Run disk virt-customize / virt-sysprep", func() {
 				Expect(err).ShouldNot(HaveOccurred())
 			}
 
-			runner.NewTaskRunRunner(f, config.GetTaskRun()).
+			r := runner.NewTaskRunRunner(f, config.GetTaskRun()).
 				CreateTaskRun().
-				ExpectFailure().
-				ExpectLogs(config.GetAllExpectedLogs()...).
+				ExpectFailure()
+			// debug lines, which should help to debug flaky tests
+			err := r.GetError()
+			if err != nil {
+				r.PrintTektonInfo()
+				if dv := config.TaskData.Datavolume; dv != nil {
+					r.PrintDatavolumeInfo(config.TaskData.Datavolume.Namespace, config.TaskData.Datavolume.Name)
+				}
+				Expect(err).ShouldNot(HaveOccurred())
```

Comment:
When `err != nil` this will always fail?

Reply:
yes, because if there is an error, it will anyway fail in later steps

Reply:
Call `Fail()` instead to make it obvious failing is intended if this line is reached?